sdk/cosmosdb/cosmos/.azure-pipelines/cosmos.test.yml

```diff
@@ -0,0 +1,55 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+trigger:
+  branches:
+    include:
+    - master
+    - cosmos-build-master
```

Comment:
Why does the branch list include `cosmos-build-master`?  Maybe it is for testing the PR, and it will be removed before merging the final PR?

Reply:
Yes, currently I configured the build to run against my fork. It will be removed.

Reply:
Removed the branch on my fork